includes/integration.php

```diff
@@ -375,8 +375,9 @@ public function sync_orders_chunk($specific_order_ids = false){
 
                             // fetch image URL
                             $image_id = get_post_thumbnail_id($product['product_id']);
-                            $image = get_post($image_id);
-                            if($image && $image->guid) $product_hash['image_url'] = $image->guid;
+                            
+                            $image_url = wp_get_attachment_image_src($image_id, 'full')[0];
```

Comment:
can you somehow check the availability of this function in old WP/WC versions?

Reply:
Compatible back to wordpress 2.5.0 (year 2008).